frontend/app/utils/search.ts

```
@@ -76,35 +93,46 @@ export class JsonUrlConverter {
+        addFilterParams(f, `${prefix}${this.keyMap.filters}[${i}].`)
+      );
     };
 
-    json.filters.forEach((filter, index) => addFilterParams(filter, `${this.keyMap.filters}[${index}].`));
-
-    const rangeValues = this.getDateRangeValues(json.rangeValue, json.startDate, json.endDate);
+    json.filters.forEach((filter: any, index: number) =>
+      addFilterParams(filter, `${this.keyMap.filters}[${index}].`)
+    );
 
     params.append(this.keyMap.rangeValue, json.rangeValue);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)